packages/docusaurus-plugin-content-docs/src/globalData.ts

```diff
@@ -20,6 +25,29 @@ export function toGlobalDataDoc(doc: DocMetadata): GlobalDoc {
   };
 }
 
+export function toGlobalSidebars(
+  sidebars: Sidebars,
+  version: LoadedVersion,
+): Record<string, GlobalSidebar> {
+  const {getFirstDocOfSidebar} = createSidebarsUtils(sidebars);
+  return mapValues(sidebars, (sidebar, sidebarId) => {
+    const firstDoc = getFirstDocOfSidebar(sidebarId);
+    if (!firstDoc) {
+      throw new Error(
+        `The sidebar "${sidebarId}" doesn't seem to contain any documentation`,
+      );
```

Comment:
wonder if it's necessary to throw here 🤷‍♂️ If the user does not use the new navbar item type, it should be fine even if the sidebar is empty. 

It's worth double-checking, as I'm not sure we support empty sidebars well already.

Is it good DX if the user is using auto-generated sidebars (default config) and has an empty `docs` folder? I think it should be allowed: the user may temporarily delete all the docs folder content to replace it with real content (I've seen an user doing exactly that in a twitch live stream recently: he just wanted to cleanup our init template)

Reply:
@slorber so I assume that we want to just send an undefined global sidebar link to the clinet-side, and then the navbar item on the client-side will check if the sidebar link is empty and throw the error there?

Reply:
I'm also a "no" on this point. Why would a user want to keep the dev server running while cleaning up the init template? What would she be trying to track?

Reply:
I've seen this in action 🤷‍♂️ users expect to be able to do complex refactors with the server running

But not a big deal, we can see later if it's useful to be more failsafe

Reply:
Also BTW, once the dev server starts running, nothing can stop it. So throwing an error isn't very bad DX since users can still work on fixing it while keeping the dev server up.

Reply:
Also BTW, once the dev server starts running, nothing can stop it. So throwing an error isn't very bad DX since users can still work on fixing it while keeping the dev server up.